### PR TITLE
Prepare the 0.1.10 release.

### PR DESCRIPTION
**Pull request by jsirois**
Provide instructions for use, updating and bug reporting.

Work towards #7

### CODE REVIEW

**Comment by jsirois** [comment]
I'll do the release of 0.1.10 this afternoon and announce in #general or #announce - not sure what makes sense. I just want to get folks hammering it.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
...
+  fetch a CPython 3.8 interpreter to use and a CPython 3.9 interpreter otherwise. These interpreters
+  are self-contained from the [Python Build Standalone](
+  https://python-build-standalone.readthedocs.io/en/latest/) project.
+
++ The ability to run Pants in a subdirectory of your project:
+
+  This is of limited utility since Pants internals don't support this well at the moment, but as
+  soon as they do, `scie-pants` will allow you to work in the style you prefer.
+
++ Built in ability to set up a new Pants project:
```
Comment (stuhood) [inline README.md:52]:
```suggestion
+ Built-in ability to set up a new Pants project:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,100 @@
...
+
+  If you run `scie-pants` in a directory where Pants is not already set up, it will prompt you, and
+  you can let it set up the latest Pants stable version for your project.
+
+## Caveats
+
+The `scie-pants` binary will re-install versions of Pants you have already installed. The underlying
+[`scie`](https://github.com/a-scie/jump/blob/main/README.md) technology uses an `nce` cache
```
Comment (stuhood) [inline README.md:60]: First working google result for "nce cache" is https://www.moko.ru/moko/nce.html ... should this link to something else explaining the acronym?

**Review by stuhood — APPROVED** [review]
Thanks!

File: README.md (hunk 2 above)
Comment (jsirois) [reply README.md:60]: I mean, I already link to the authoritative source in the very same sentence ...

Comment (stuhood) [reply README.md:60]: Hm. That page does not define `nce`... is it just the suffix of `science`, and not supposed to mean anything else? It _looks_ like an acronym.

Comment (jsirois) [reply README.md:60]: It does. At base it's just a name for a cache dir explained here: https://github.com/a-scie/jump/blob/b5b81a611d482f572d5369731b70279f5579c597/README.md?plain=1#L66-L70

And if you read out the last words of the last sentence its also explained.

Comment (jsirois) [reply README.md:60]: You well know my disdain for words. I'm very happy to review doc edit PRs that make things make sense to you and others.

**Pull request merged**